main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+    "database/sql"
+    "encoding/csv"
+    "fmt"
+    "net/http"
+    "os"
+    "sync/atomic"
+   "time"
+   _ "github.com/mattn/go-sqlite3"
+)
+
+var counter uint64
+
+func main() {
+    db, err := sql.Open("sqlite3", "./requests.db")
+    if err != nil {
+        fmt.Println(err)
+        return
+    }
+    defer db.Close()
+
+    _, err = db.Exec("CREATE TABLE IF NOT EXISTS requests (id INTEGER PRIMARY KEY, timestamp DATETIME DEFAULT CURRENT_TIMESTAMP)")
+    if err != nil {
+        fmt.Println(err)
+        return
+    }
+
+    go func() {
+        for {
+            <-time.After(time.Minute)
+            count := atomic.LoadUint64(&counter)
+            fmt.Printf("Total requests received: %d\n", count)
+            _, err := db.Exec("INSERT INTO requests (timestamp, count) VALUES (CURRENT_TIMESTAMP, ?)", count)
```

Comment:
The requests table doesn't have the count column. You also need to change Line 24 to add a count column.